plugins/pixart-rf/fu-pxi-firmware.h

```diff
@@ -17,3 +17,9 @@ FuFirmware *
 fu_pxi_firmware_new(void);
 const gchar *
 fu_pxi_firmware_get_model_name(FuPxiFirmware *self);
+
+void
+fu_pxi_firmware_set_is_hpac_flag(FuPxiFirmware *self, gboolean flag);
+
+gboolean
+fu_pxi_firmware_get_is_hpac_flag(FuPxiFirmware *self);
```

Comment:
missing newline character at end of file

Reply:
solved in a3f8969ce1bb3ddf34e463aa232dc6861f6f6d3e